src/main/java/com/amplitude/Amplitude.java

```diff
@@ -136,12 +95,24 @@ private Object syncHttpCall(Event event) {
                 sb.append(output);
             }
 
-            if (responseCode >= 400) {
-                System.err.println("Warning, received error " + responseCode + " with message: " + sb.toString());
+            if (responseCode >= Constants.BAD_RES_CODE_START) {
+                AmplitudeLog.log(TAG, "Warning, received error HTTP code " + responseCode + " with message: " + sb.toString(),
+                        AmplitudeLog.LogMode.WARN);
+            }
+            else {
+                AmplitudeLog.log(TAG, "Successful HTTP code " + responseCode + " with message: " + sb.toString(),
```

Comment:
User can't control the logging info. What if they don't wanna see the debug info at all? 

You need to have a log level api exposed to the users to control the logging info.